code/modules/surgery/other.dm

```diff
@@ -101,6 +111,69 @@
 
 	return SURGERY_STEP_RETRY
 
+/datum/surgery_step/fix_vein/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	user.visible_message(
+		"<span class='warning'> [user]'s hand slips, smearing [tool] in the incision in [target]'s [affected.name]!</span>",
+		"<span class='warning'> Your hand slips, smearing [tool] in the incision in [target]'s [affected.name]!</span>"
+	)
+	affected.receive_damage(5, 0)
+
+	return SURGERY_STEP_RETRY
+
+/datum/surgery_step/treat_burns
+	name = "mend burns"
+	allowed_tools = list(
+		/obj/item/stack/medical/ointment/advanced = 100,
+		/obj/item/stack/medical/ointment = 90
+	)
+	can_infect = TRUE
+	blood_level = SURGERY_BLOODSPREAD_HANDS
+
+	time = 3.2 SECONDS
+
+/datum/surgery_step/treat_burns/begin_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	if(!(affected.status & ORGAN_BURNT))
+		to_chat(user, "<span class='warning'>The skin on [target]'s [parse_zone(affected)] seem to be in perfect condition, it doesn't need treatment.</span>")
+		return SURGERY_BEGINSTEP_SKIP
+
+	user.visible_message(
+		"[user] starts to treat affected tissue in [target]'s [affected.name] with \the [tool].",
+		"You start to treat affected tissue in [target]'s [affected.name] with \the [tool]."
+	)
+	target.custom_pain("The pain in your [affected.name] is unbearable!")
+
+	return ..()
+
+/datum/surgery_step/treat_burns/end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	var/obj/item/stack/stack = tool
+
+	stack.use(1)
+	affected.fix_burn_wound()
+	affected.germ_level = 0
+	target.update_body()
+
+	user.visible_message(
+		"<span class='notice'> [user] finishes treating affected tissue in [target]'s [affected.name]</span>",
+		"<span class='notice'> You finish treating affected tissue in [target]'s [affected.name] with \the [tool].</span>"
+	)
+
+	return SURGERY_STEP_CONTINUE
+
+/datum/surgery_step/treat_burns/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+	var/obj/item/stack/stack = tool
+
+	stack.use(1)
+
+	user.visible_message(
+		"<span class='warning'> [user]'s hand slips, ineffectively treating [target]'s [affected.name] with the [tool]!</span>",
+		"<span class='warning'> Your hand slips, ineffectively treating [target]'s [affected.name] with the [tool]!</span>"
```

Comment:
```suggestion
		"<span class='warning'> [user]'s hand slips, applying \the [tool] in the wrong place on [target]'s [affected.name]!</span>",
		"<span class='warning'> Your hand slips, applying \the [tool] in the wrong place on [target]'s [affected.name]!</span>"
```
As above

Reply:
unnecessary space at the start too.

Reply:
Actually, just remove `the` and don't add `\the`. It's unnecessary.